src/Storage/Storage.Test/Storage.Test.csproj

```diff
@@ -14,9 +14,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="0.10.1-preview" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.Storage.File" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="10.0.3" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.0.0" />
```

Comment:
Doesn't this require changes to the Storage.DataMovement library?

Reply:
As we discussed before, since Az.Accounts reference a old DMlib, we will use the private signed DMlib in PSH. And I have already updated the private signed DMlib to new build with this PR.